google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryDryRunResult.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.bigquery;
+
+import com.google.api.services.bigquery.model.QueryParameter;
+import java.util.List;
+
+public interface BigQueryDryRunResult {
+
+  /** Returns the schema of the results. Null if the schema is not supplied. */
+  Schema getSchema() throws BigQuerySQLException;
+
+  /** Returns query parameters for standard SQL queries */
+  List<QueryParameter> getQueryParameters() throws BigQuerySQLException;
```

Comment:
I'm assuming this is a return of the undeclared query params to determine type info?  Or is this returning the params that were provided as part of the dry run?

Reply:
This should be returning the undeclared query params to determine type info (the former).